client/my-sites/plugins/not-available-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NotAvailableError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
+				<SidebarNavigation />
+				<EmptyContent
+					title={ translate( 'Not Available' ) }
+					line= { translate( 'The page you requested could not be found' ) }
```

Comment:
@tyxla should this be an error message or should we show the sites list here with a message saying 'to manage plugins first select a site'? If they only have one site what would happen in that case I wonder? If we do choose to go with an error message here I think we do need some more information to draft the appropriate copy (like @michelleweber hinted at). I can't entirely tell if this PR refers to a situation where the Manage module is disabled or if the user's role is inadequate. If manage is disabled, we should invite the user to enable it. If their role permissions are the issue, we should kindly let them know. I guess I would just need more info to help draft a nuanced message.